spatial-tracking-explainer.md

```diff
@@ -318,18 +318,6 @@ function onSessionStarted(session) {
 }
 ```
 
-While many sites will be able to provide this fallback, for some sites this will not be possible.  Under these circumstances, it is instead preferable for session creation to reject rather than spin up immersive display/tracking systems only to immediately exit the session.
-
-```js
-function beginImmersiveSession() {
-  xrDevice.requestSession({ immersive: true,  requiredReferenceSpaceType:'unbounded' })
```

Comment:
So this sample code made it into the explainer, but never made its way into the spec because there was confusion about the overall permissions/features model.  While we are working through the final shape of that design (oh, John..... *poke*poke*poke*) can we at least file an issue to cover adding this functionality back in once that's complete?

Reply:
Alternatively, we can leave the options dictionary and put this in it ;)